### ISSUE TITLE
[ERROR] Field does not exist on enclosing type WHEN USING IN filtering

### ISSUE THREAD

**Issue by rihellori** [prisma/prisma#3465]
## Bug description

I want to get chat data from filtering players' IDs.
When I used prisma version 1.

My code works fine like below.

`let chat = await prisma.chats({  where: { players_every: { id_in: playerIds } }    });
`====> It works well!

BUT, I changed my code to prisma2.
It does not work.
model which I make is same to prisma version 1.

// In JS file code
`await prisma.chat.findMany({ where: { players: { some: { id: { in: playerIds } } } } });
`===> IT shows error

 ```
Failed to validate the query `Error occurred during query validation & transformation:",
            "Query (object)",
            "  ↳ findManyChat (field)",
            "    ↳ where (argument)",
            "      ↳ ChatWhereInput (object)",
            "        ↳ players (field)",
            "          ↳ PlayerListRelationFilter (object)",
            "            ↳ is (field)",
            "              ↳ Field does not exist on enclosing type.` at `.Query.findManyChat.where.ChatWhereInput.players.PlayerListRelationFilter.is`",
"    at _callee2$ (/Users/rio/Devel/jelato-backend/src/Gapi/Chat/ReadChat/ReadChat.js:38:15)",
            "    at tryCatch (/Users/rio/Devel/jelato-backend/node_modules/regenerator-runtime/runtime.js:45:40)",
            "    at Generator.invoke [as _invoke] (/Users/rio/Devel/jelato-backend/node_modules/regenerator-runtime/runtime.js:274:22)",
            "    at Generator.prototype.(anonymous function) [as throw] (/Users/rio/Devel/jelato-backend/node_modules/regenerator-runtime/runtime.js:97:21)",
            "    at asyncGeneratorStep (/Users/rio/Devel/jelato-backend/src/Gapi/Chat/ReadChat/ReadChat.js:10:103)",
            "    at _throw (/Users/rio/Devel/jelato-backend/src/Gapi/Chat/ReadChat/ReadChat.js:12:291)",
            "    at process._tickCallback (internal/process/next_tick.js:68:7)"
```

what is field "is" ????

## Prisma information

```
model Chat {
  id           String     @default(cuid()) @id
  players      Player[]
  createdAt    DateTime   @default(now())
  updatedAt    DateTime   @updatedAt
}

model Player {
  id           String     @default(cuid()) @id
  chats        Chat[]
  createdAt    DateTime   @default(now())
  updatedAt    DateTime   @updatedAt
}

```
`await prisma.chat.findMany({ where: { players: { some: { id: { in: playerIds } } } } });
`===> It does not work.

## Environment & setup

```
@prisma/cli          : 2.5.1
Current platform     : darwin
Query Engine         : query-engine c88925ce44a9b89b4351aec85ba6a28979d2658e (at ../../../../usr/local/lib/node_modules/@prisma/cli/query-engine-darwin)
Migration Engine     : migration-engine-cli c88925ce44a9b89b4351aec85ba6a28979d2658e (at ../../../../usr/local/lib/node_modules/@prisma/cli/migration-engine-darwin)
Introspection Engine : introspection-core c88925ce44a9b89b4351aec85ba6a28979d2658e (at ../../../../usr/local/lib/node_modules/@prisma/cli/introspection-engine-darwin)
Format Binary        : prisma-fmt c88925ce44a9b89b4351aec85ba6a28979d2658e (at ../../../../usr/local/lib/node_modules/@prisma/cli/prisma-fmt-darwin)
Studio               : 0.261.0

```

**Comment by pantharshit00** [comment]
Hey @prismaLover 

Can you please try this again in the latest dev version `npm install @prisma/cli@dev @prisma/client@dev`

We had a similar bug fixed recently. Don't forget to run prisma generate after you update.

**Comment by prismaLover** [comment]
Thanks so much!

It works with @prisma/cli@dev and @prisma/client@dev

Your reply is so fast lol!

when is this function applied to main version. and what version ???

PRISMA FOREVER!

**Comment by pantharshit00** [comment]
Thanks for the kind words 😁

The fix should actually be already in `2.5.1` but you might have forgotten to regenerate the client.

Nevertheless release `2.6.0` is planned for today so you can check that out as well.
